### PR TITLE
Expand result table to full available width

### PR DESCRIPTION
**Pull request by Mexolius**

### CODE REVIEW

File: src/Components/ResultTable/ResultTable.js [hunk 1]
```diff
@@ -2,9 +2,11 @@ import React, { useMemo } from 'react'
 import { useTable, useSortBy } from 'react-table'
 
 import './ResultTable.css'
-import { getCSV } from './TableUtils'
+import { getInvisibleStyle } from './TableUtils'
 
-function ResultTable({ energies, values, stoppingPowerUnit }) {
+// Once #
```
Comment (grzanka) [inline src/Components/ResultTable/ResultTable.js:7]: What does it mean ?

**Comment by grzanka** [comment]
Works well, but sorting by energy in descending order is broken.

shot-2022-01-02_17-26-59

File: src/Components/ResultTable/ResultTable.js (hunk 1 above)
Comment (Mexolius) [reply src/Components/ResultTable/ResultTable.js]: Comment to deleted function. Removed it.

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**